tests/testutil/router.go

```diff
@@ -62,6 +66,40 @@ func (tr *TestRoute) GetEndpoint() string {
 	return response
 }
 
+// GetKubeEndpoints performs an http.get against the route's full domain and all paths on it
+// Returns responses if status code is 200 for all of them, otherwise it errors out
+func (tr *TestRoute) GetKubeEndpoints() []string {
+	var responses []string
+
+	// Get Router object using kubectl
+	args := []string{"get", "routers", tr.Router.GetName(), "-n", testingNamespace, "-o", "json"}
+	resultString, err := KubectlCmd(args)
+	if err != nil {
+		tr.T.Fatalf("Failed to get rio.cattle.io.routers:  %v", err.Error())
+	}
+	var results riov1.Router
+	err = json.Unmarshal([]byte(resultString), &results)
+	if err != nil {
+		tr.T.Fatalf("Failed to unmarshal results:  %v", err.Error())
+	}
+
+	// Validate there is a base endpoint
+	if len(results.Status.Endpoints) == 0 {
+		tr.T.Fatal("router has no endpoint")
+	}
+
+	// Store response from URLs of each path on the route
+	for _, path := range results.Spec.Routes {
+		res, err := WaitForURLResponse(fmt.Sprintf("%s%s", results.Status.Endpoints[0], path.Matches[0].Path.Exact))
+		if err != nil {
+			tr.T.Fatal(err.Error())
+		}
+		responses = append(responses, res)
```

Comment:
I wonder if we should make this method single route based to match `GetEndpoint`, you just range through results till you get your wanted path. This is part of larger discussion around the tricky design of TestRouter and multiple paths.

Reply:
Yeah we can handle this the same way as the other change. Keep it simple. 2 routes, they'll both have the same domain, just do what `GetEndpoint(Response)` does with tr.Path and call `KubeGetEndpoint(Response)` on each separate route